ReactCommon/fabric/components/text/paragraph/ParagraphState.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "ParagraphState.h"
+
+#include <react/components/text/conversions.h>
+#include <react/debug/debugStringConvertibleUtils.h>
+
+namespace facebook {
+namespace react {
+
+AttributedString ParagraphState::getAttributedString() const {
+  return attributedString_;
+}
+
+void ParagraphState::setAttributedString(const AttributedString &attributedString) const {
+  assert(!attributedStringIsInitialized_);
```

Comment:
this assert gets called after pushing a new view in RNTester, so maybe I am not using this correctly?